docs/_includes/links.md

```diff
@@ -3,13 +3,14 @@
 <!-- Concepts -->
 
 [Fluid container]: {{< relref "containers.md" >}}
+[Signals]: {{< relref "/docs/concepts/signals.md" >}}
 
 <!-- Distributed Data Structures -->
 
 [SharedCounter]: {{< relref "/docs/data-structures/counter.md" >}}
 [SharedMap]: {{< relref "/docs/data-structures/map.md" >}}
-[SharedSequence]: {{< relref "/docs/data-structures/sequences.md" >}}
 [SharedString]: {{< relref "/docs/data-structures/string.md" >}}
+[Sequences]:  {{< relref "/docs/data-structures/sequences.md" >}}
```

Comment:
It doesn't look like we're using this anywhere. Should we remove it?

Reply:
The doc is still published, so I'm leaving this as-is since the goal of the included links is to be able to link to common topics without thinking too hard.